src/aics-image-viewer/shared/utils/storage.ts

```diff
@@ -0,0 +1,162 @@
+import { mapValues } from "lodash";
+
+import type { MetadataRecord } from "../types";
+
+const QUEUE_KEY = "GLOBAL_ENTRY_QUEUE";
+
+const enum StorageEntryType {
+  Scenes = "scenes",
+  Meta = "meta",
```

Comment:
Nit: Enum values conventionally all caps

Reply:
Seeing both styles in this repo, but mainly this one